components/worktemplates/components/menu.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+import styled from 'styled-components';
+
+import {useIntl} from 'react-intl';
+
+import classNames from 'classnames';
+
+import {Category, WorkTemplate} from '@mattermost/types/worktemplates';
+import {worktemplates as workTemplateDevData} from '../dev_data';
```

Comment:
nit: try to organize imports by context, it is really helpful when making code archeology